tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"declaration": true,
+		"emitDeclarationOnly": true,
```

Comment:
Do we need these? We’re not ever building with `tsc` so I would have thought emitting `.d.ts` files was not really relevant.

Reply:
I have actually used `tsc` a couple of times already as kind of a TS linter. You can just run it from the repo root and it will output all issues from all scripts instead of having to go through them manually in VS Code. If you remove those options, it's not possible to run `tsc` anymore and it will error.